FirstTask.py

```diff
@@ -0,0 +1,180 @@
+from os import system as sys
+
+class BankAccount:
+    def __init__(self, Name, PassCode, AccNumber, Balance, AccoutType):
+        self.Name = Name
+        self.Passcode = PassCode
+        self.Balance = Balance
+        self.AccountType = AccoutType
+        self.AccNumber = AccNumber
+
+    def FullName(self):
+        return self.Name
+
+    def AccountNumber(self):
+        return self.AccNumber
+
+    def PassCode(self):
+        return self.Passcode
+
+    def CheckBalance(self):
+        return self.Balance
+
+    def CheckAccountType(self):
+        return self.AccountType
+
+    def AccDetails(self):
+        print("Account # ", self.AccNumber)
+        print("Name: ", self.Name)
+        print("Balance: ", self.Balance)
+        print("Account Type: ", self.AccountType)
+
+    def DepositBill(self,Bill):

```

Comment:
A space is required after the comma here along with the name change to `deposit_bill`. Also, the param should be `bill`.